### PR TITLE
[ROCKETMQ-32] Improve concision - Reuse local variable 'brokerAddrs' in RouteInfoManager.getSystemTopicList method

### PR DESCRIPTION
**Pull request by naughtybear**
reuse brokerAddrs, or change like following:
`    if (bd.getBrokerAddrs() != null && !bd.getBrokerAddrs().isEmpty()) {
        Iterator<Long> it2 = bd.getBrokerAddrs().keySet().iterator();
        topicList.setBrokerAddr(bd.getBrokerAddrs().get(it2.next()));
        break;
    }`

### CODE REVIEW

**Comment by zhouxinyu** [comment]
Hi @naughtybear,

Thanks your PR, could you abide our PR process(modify your PR subject and mention your jira address in the description), please follow this PR. #5

**Comment by naughtybear** [comment]
Hi @zhouxinyu ,

Sorry.
I just start learning and wanna contributing, it's very nice of you to guide me.

Code concision improvement.
JIRA Issue: https://issues.apache.org/jira/browse/ROCKETMQ-11

**Comment by vongosling** [comment]
please modify you PR subject and fix conflict problems.

**Comment by naughtybear** [comment]
@vongosling 

All done.

**Comment by vongosling** [comment]
Subject still is "simplify if grammar "?please bind with JIRA. You can refer to other PR:-)

**Comment by vongosling** [comment]
Could you polish again just like other PR. such as title start with JIRA issues. The following links may be helpful for you. 

http://rocketmq.incubator.apache.org/docs/pull-request/
http://www.bigcloud.online/?p=280

**Comment by vongosling** [comment]
Yes, do not start a new pull request.Just modify your topic, start a JIRA and bind them :-)

**Comment by zhouxinyu** [comment]
It seems ok, thanks @naughtybear .

Please @vongosling @lizhanhui help review it.

**Comment by naughtybear** [comment]
@vongosling @lizhanhui 
Please help review. Thanks.

File: rocketmq-namesrv/src/main/java/com/alibaba/rocketmq/namesrv/routeinfo/RouteInfoManager.java [hunk 1]
```diff
@@ -622,7 +622,7 @@ public void printAllPeriodically() {
                     while (it.hasNext()) {
                         BrokerData bd = brokerAddrTable.get(it.next());
                         HashMap<Long, String> brokerAddrs = bd.getBrokerAddrs();
-                        if (bd.getBrokerAddrs() != null && !bd.getBrokerAddrs().isEmpty()) {
```
Comment (vongosling) [inline rocketmq-namesrv/src/main/java/com/alibaba/rocketmq/namesrv/routeinfo/RouteInfoManager.java]: code smell

**Comment by vongosling** [comment]
alright

**Pull request closed without merging**